dynoid/middleware/example_test.go

```diff
@@ -0,0 +1,24 @@
+package middleware_test
+
+import (
+	"io"
+	"log"
+	"net/http"
+
+	"github.com/heroku/x/dynoid/middleware"
+)
+
+const AUDIENCE = "testing"
```

Comment:
I think [constants should be mixed caps](https://google.github.io/styleguide/go/decisions.html#constant-names) although maybe this is to signal that you replace it?